xl/playlist.py

```diff
@@ -2115,6 +2115,11 @@ def save_playlist(self, pl, overwrite=False):
                 playlist should it happen to already exist
         """
         name = pl.name
+
+        # Make sure name is unicode
+        if not isinstance(name, unicode):
```

Comment:
Is it possible for `pl.name` to be an str? If it's always unicode we can just `assert(isinstance(name, unicode))` here, which will help us clarify the types in the future when moving to Python 3.